src/app/question-authoring-page/question-authoring-page.component.html

```diff
@@ -0,0 +1,64 @@
+<clr-main-container class="main-container">
+<clr-header class="header header-1">
+    <div class="branding">
+        <a routerLink="/home"  routerLinkActive="active" class="nav-link">
+            <img src="assets/dojo-transparent-white-outline.svg" alt="A dojo" class="mtdj__logo">
+            <span class="title">Math Dojo</span>
+        </a>
+    </div>
+
+</clr-header>
+<div class=" question-form-wrapper" >
+    <form  class="question-form" [formGroup]="newQuestionForm" (submit)="onSubmit()">
+        <section class="title">
+            <h3 class="welcome">Math Dojo Question Creation Page</h3>
+            
+        </section>
+        <br>
+           
+                <label class="clr-sr-only">Question Title</label>
+                <input type="text" name="title" clrInput placeholder="Question Title" formControlName="title"/>
+                <p style="color:red" *ngIf="!newQuestionForm.controls.title.valid && submitted">Required</p>
+          <br>
+                <label >Question Body</label><br>
+                <textarea style="height:200px;width: 900px"  type="text" name="body" formControlName="body"name="Text1" cols="40" rows="5"></textarea>
+                <p style="color:red" *ngIf="!newQuestionForm.controls.body.valid && submitted">Required</p>
+                <br> 
+                <label >Sample Answer</label><br>
+                <textarea style="height:100px;width: 900px" type="text" name="answer"cols="200" rows="5"  formControlName="answer"></textarea>
+                <br>
+           
+                <label class="clr-sr-only">Hints</label>
+                <input type="text" name="hints" clrInput placeholder="Hints (Maximum 3 seperated by semicolons)" formControlName="hints"/>
+                <p style="color:red" *ngIf="!newQuestionForm.controls.hints.valid && validateHints">Must be max of 3 Hints seperated by semicolons</p>
+                <br>
+
+                <label >Difficulty</label><br>
+                <input type="text" name="difficulty" list="difficultyOptions">
+                <datalist id="difficultyOptions">
+                    <option>Easy</option>
+                    <option>Medium</option>
+                    <option>Hard</option>
+                </datalist> 
+            <br>
+            <br>
+            <label class="clr-sr-only">Options</label>
+                <input type="text" name="options" clrInput placeholder="Options (Maximum 4 seperated by semicolons)" formControlName="options"/>
+                <p style="color:red" *ngIf="!newQuestionForm.controls.options.valid && validateOptions ">Must be max of 4 options seperated by semicolons</p>
+                <br>
+                <br>
+                <label class="clr-sr-only">Topic Title</label>
+                <input type="text" name="parentTopicTitle" clrInput placeholder="Topic Title" formControlName="parentTopicTitle"/>
```

Comment:
I think this would be a good candidate for a data list. Reason being that we wouldn’t want to give our users the ability to make topics on-the-fly (this has implications in terms of the level of checking we’d need to do at quarantine). On page load, the authoring page could make a call to find out what the list of possible topics is and then provide those as options a user could choose from (perhaps as they start to type even).